link error when using  MakeSpanContext function

when we use the   MakeSpanContext function 
https://github.com/lightstep/lightstep-tracer-cpp/blob/939e8d7dd946e37e18b763553c98724244657654/include/lightstep/tracer.h#L158
we got link errors. 
We want to create our own spancontext by the traceid and spanid.

Here it is. 
```
Use --sandbox_debug to see verbose messages from the sandbox
atlas/commands/cmdbase_atlas.cpp:160: error: undefined reference to 'lightstep::LightStepTracer::MakeSpanContext(unsigned long, unsigned long, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > >&&) const'
collect2: error: ld returned 1 exit status
```
I switched to the ``` opentracing::expected<std::unique_ptr<opentracing::SpanContext>>
  MakeSpanContext(uint64_t trace_id, uint64_t span_id, bool sampled,
                  std::unordered_map<std::string, std::string>&& baggage) const
      noexcept;
``` and it works.

--- a/include/lightstep/tracer.h
+++ b/include/lightstep/tracer.h
@@ -157,7 +157,9 @@ class LightStepTracer : public opentracing::Tracer {
   opentracing::expected<std::unique_ptr<opentracing::SpanContext>>
   MakeSpanContext(uint64_t trace_id, uint64_t span_id,
                   std::unordered_map<std::string, std::string>&& baggage) const
-      noexcept;
+      noexcept {
+    return this->MakeSpanContext(trace_id, span_id, true, std::move(baggage));
+  }
 
   opentracing::expected<std::unique_ptr<opentracing::SpanContext>>
   MakeSpanContext(uint64_t trace_id, uint64_t span_id, bool sampled,

